pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/DimensionStats.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.stats;
+
+import org.HdrHistogram.Histogram;
+import org.HdrHistogram.Recorder;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ */
+public class DimensionStats {
+
+    /** Statistics for dimension times **/
+    public double meanDimensionMs;
+
+    public double medianDimensionMs;
+
+    public double dimension95Ms;
+
+    public double dimension99Ms;
+
+    public double dimension999Ms;
+
+    public double dimension9999Ms;
+
+    public double dimensionCounts;
+
+    public double elapsedIntervalMs;
+
+    private Recorder dimensionTimeRecorder = new Recorder(TimeUnit.MINUTES.toMillis(10), 2);
```

Comment:
Instead of using the `HdrHistogram` here, we could use the Prometheus client lib. It has the same support for the quantiles and it will get automatically exposed and reported in the `/metrics` REST handler